cli/src/pcluster/utils.py

```diff
@@ -920,12 +920,12 @@ def get_info_for_amis(ami_ids):
         error(e.response.get("Error").get("Message"))
 
 
-def get_ami_id(parent_image):
-    """Get ami id from parent image, parent image could be image id or image arn."""
-    if parent_image.startswith("arn"):
-        ami_id = ImageBuilderClient().get_image_id(parent_image)
+def evaluate_ami_id(ami):
```

Comment:
-minor: I like more the previous naming `get_ami_id(parent_image)`

Reply:
This change has been done after my comment :D 

The _get_ word isn't clear when you're already passing an ami id to the method. I mean, getting an ami id from an ami id?

The _get_ word let me think we need to obtain the value from somewhere, _evaluate_ (or _determine_, or another word) let me think we're evaluating it from the given value.

Reply:
we are getting the ami id from the `parent_image` string, that could be either an `image arn` or an `ami id` already. In case it's an `image arn` we are calling an api to retrieve the `ami id`.

Reply:
I create a `imagebuilder_utils` module in `common` dir to put imagebuilder related utility functions, and move `get_ami_id` to that module